contracts/clients/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bls-wallet-clients",
-  "version": "0.7.3",
+  "version": "0.8.0",
```

Comment:
@voltrevo Given contract changes, especially signed payload format w/ wallet address i bumped the minor to indicate a breaking change.